doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories.
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects). The latter step creates the `DECam/calib` collection.
+
+.. prompt:: bash
+
+   cd /path/to/desired/repo
+   butler create .
+   butler register-instrument . lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations . lsst.obs.decam.DarkEnergyCamera
+
+Ingest raw science and calibration (bias and flat) frames, which creates the `DECam/raw/all` collection. Run overscan correction on all the raws so they may be used as crosstalk sources during ISR.
```

Comment:
It's a little hard to attach these sentences with the commands they are supposed to be associated with. I'm not sure of a better approach, though, other than having each sentence be directly above the associated command (and thus having many more `.. prompt::` sections, which may not be ideal.

Some of the sentences almost look like section headings? Maybe having sections to group things would help?

Reply:
I will add some subsection headings